scripts/cmd_test/aes-cmd-test.sh

```
@@ -33,19 +33,29 @@ source "${UTILS_DIR}/utils-openssl.sh"
 source "${UTILS_DIR}/utils-wolfssl.sh"
 source "${UTILS_DIR}/utils-wolfprovider.sh"
 
-# Initialize the environment
-init_wolfprov
+# Initialize wolfProvider without WPFF set
```

Can you elaborate on why the force fail flag needs to be unset for the `init_wolfprov` call?

---

Yes, If you set the force fail variable before you build the unit test will fail and cause the cmd test to fail. unseting force fail for the build makes sure this doesn't happen.